apps/cowswap-frontend/src/modules/tradeWidgetAddons/containers/TradeRateDetails/index.tsx

```diff
@@ -50,51 +44,21 @@ export function TradeRateDetails({ rateInfoParams, deadline, isTradePriceUpdatin
   const derivedTradeState = useDerivedTradeState()
   const tradeQuote = useTradeQuote()
   const shouldPayGas = useShouldPayGas()
-  // const { isSmartContractWallet } = useWalletDetails()
-
-  // TODO(bridge): Bridge-related hooks and state
-  // TODO(bridge): Use the bridge hooks and state from the bridge module once the bridge is implemented
-  // const _isBridgingEnabled = useIsBridgingEnabled(isSmartContractWallet)
   const isHooksTabEnabled = useIsHooksTradeType()
+  const { isSmartContractWallet } = useWalletDetails()
 
-  // TODO(bridge): For now, force showBridgeUI to true for demo purposes.
-  // In production, this should be: const showBridgeUI = _isBridgingEnabled && _isCurrentTradeBridging
-  const showBridgeUI = true
-
-  // TODO(bridge): Use default bridge provider for now - can be made configurable later
-  const bridgeProvider = BridgeProvider.BUNGEE
-  const providerDetails = useBridgeProviderDetails(bridgeProvider)
-
-  // Create example bridge data using the provider details
-  const bridgeData = useMemo(() => {
-    if (!showBridgeUI) return null
-
-    try {
-      const data = createBridgeData({
-        bridgeProvider: providerDetails,
-        estimatedTime: getBridgeEstimatedMinutes(),
-      })
-
-      return data
-    } catch (error) {
-      console.error('Failed to create bridge data:', error)
-      // In case of error, return null to fall back to regular trade UI
-      return null
-    }
-  }, [providerDetails, showBridgeUI])
-
-  // Get bridge time estimate with error handling
-  const bridgeEstimatedTime = useMemo(
-    () => (showBridgeUI && bridgeData ? bridgeData.estimatedTime : undefined),
-    [showBridgeUI, bridgeData],
-  )
+  // Bridge related state
+  const isBridgingEnabled = useIsBridgingEnabled(isSmartContractWallet)
+  const isCurrentTradeBridging = useIsCurrentTradeBridging()
+  const showBridgeUI = isBridgingEnabled && isCurrentTradeBridging
+  const bridgeData: BridgeData | null = null
```

Comment:
Please keep a TODO here in order to set a real vaue for bridgeData in the future